### PR TITLE
Update rules

### LINKED ISSUE #686: kube-prometheus: Mismatch between dashboards and rules
https://github.com/coreos/prometheus-operator/blob/9d4f8feb906d465008a0ea45fb94782c3f90446a/contrib/kube-prometheus/assets/prometheus/rules/kubernetes.rules#L134 has `cluster_resource_verb:apiserver_latency:quantile_seconds`

https://github.com/coreos/prometheus-operator/blob/9d4f8feb906d465008a0ea45fb94782c3f90446a/contrib/kube-prometheus/assets/grafana/kubernetes-control-plane-status-dashboard.json#L401 references `apiserver_latency_seconds:quantile`

I can't find a definition for `apiserver_latency_seconds:quantile`, so the dashboard doesn't work.

(Similar issue for all the graphs that aren't `...UP` in that dashboard)

### PR DESCRIPTION
**Pull request by brancz**
Closes #686

@ant31

### CODE REVIEW

**Comment by jason-riddle** [comment]
After running `hack/cluster-monitoring/minikube-deploy`, the following is shown.

```
The ServiceMonitor "alertmanager" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "kube-apiserver" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "kube-controller-manager" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "kube-scheduler" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "kube-state-metrics" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "kubelet" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "node-exporter" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "prometheus-operator" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The ServiceMonitor "prometheus" is invalid: apiVersion: Invalid value: "ServiceMonitor": must be monitoring.coreos.com/v1
The Prometheus "k8s" is invalid: apiVersion: Invalid value: "Prometheus": must be monitoring.coreos.com/v1
...
The Alertmanager "main" is invalid: apiVersion: Invalid value: "Alertmanager": must be monitoring.coreos.com/v1
```

Thoughts?

**Comment by jason-riddle** [comment]
Ah sorry, my cluster was in a weird state. Recreating it and running `minikube-deploy` works smoothly.

File: contrib/kube-prometheus/assets/prometheus/rules/kubelet.rules.yaml [hunk 1]
```diff
@@ -18,20 +18,17 @@ groups:
...
     labels:
       severity: warning
     annotations:
       description: Prometheus failed to scrape {{ $value }}% of kubelets.
-      summary: Many Kubelets cannot be scraped
   - alert: K8SKubeletDown
-    expr: absent(up{job="kubelet"} == 1) or count(up{job="kubelet"} == 0) / count(up{job="kubelet"})
-      > 0.1
+    expr: (absent(up{job="kubelet"} == 1) or count(up{job="kubelet"} == 0) / count(up{job="kubelet"}))
+      * 100 > 1
```
Comment (atoato88) [inline contrib/kube-prometheus/assets/prometheus/rules/kubelet.rules.yaml]: In previous commit, it check 10% over of `up{job="kubelet"} == 0`.
So, `expr` is below, I think.

```diff
-      * 100 > 1
+      * 100 > 10
```

If it's my misunderstanding, sorry for this comment.

Comment (atoato88) [reply contrib/kube-prometheus/assets/prometheus/rules/kubelet.rules.yaml]: @brancz Yes, I've create PR #1032 .

**Pull request merged**